karma.conf.js

```diff
@@ -49,17 +48,17 @@ module.exports = function(karma) {
 			'src/index.js': ['sources']
 		},
 		rollupPreprocessor: {
-			plugins: [
-				nodeResolve(),
-				commonjs()
-			],
 			external: [
-				'chart.js'
+				'chart.js',
+				'chart.js/helpers',
+				'chartjs-chart-financial',
```

Comment:
I'm not a karma expert. Can you add a comment explaining why this need to be `external` / global?

Reply:
It's not directly a Karma option, it comes from the rollupPreprocessor: https://rollupjs.org/guide/en/#external
AFAIU, it has to be done this way to ensure the library is not included in the karma bundle and the tests run using the UMD version of the built files, similar to how end-users would use the library if not relying on a builder.
I adopted the same pattern found in others mainstream chart.js plugins.